### PR TITLE
Add Problem 1604

### PR DESCRIPTION
**Pull request by AquaIndigo**

### CODE REVIEW

File: src/problems/1604.alert-using-same-key-card-three-or-more-times-in-a-one-hour-period.jl [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+keyName.length == keyTime.length
+keyTime[i] is in the format "HH:MM".
+[keyName[i], keyTime[i]] is unique.
+1 <= keyName[i].length <= 10
+keyName[i] contains only lowercase English letters. =#
+# @lc code=start
+using Dates
+function alert_names(key_name::Vector{String}, key_time::Vector{String})
+    res = Vector{String}()
+    mp = Dict{String, Vector{Time}}()
```
Comment (findmyway) [inline src/problems/1604.alert-using-same-key-card-three-or-more-times-in-a-one-hour-period.jl:51]: You can use [`DefaultDict`](https://juliacollections.github.io/DataStructures.jl/latest/default_dict/) in `DataStructures` here.

**Review by findmyway — APPROVED** [review]
👍 

By the way, you can rebase this PR to make it easier to review.

https://github.com/edx/edx-platform/wiki/How-to-Rebase-a-Pull-Request

File: src/problems/1604.alert-using-same-key-card-three-or-more-times-in-a-one-hour-period.jl (hunk 1 above)
Comment (AquaIndigo) [reply src/problems/1604.alert-using-same-key-card-three-or-more-times-in-a-one-hour-period.jl:51]: I know that, but I was too lazy to do so. XD

Comment (findmyway) [reply src/problems/1604.alert-using-same-key-card-three-or-more-times-in-a-one-hour-period.jl:51]: 😆

**Pull request merged**